src/BloomBrowserUI/bookEdit/js/bloomEditing.js

```diff
@@ -440,22 +440,23 @@ function MakeSourceTextDivForGroup(group) {
   else {
     $(divForBubble).remove();//no tabs, so hide the bubble
     return;
-  }
+    }
+
+    var showEvents = false;
+    var hideEvents = false;
+    var shouldShowAlways = true;
+
+  
+    if(checkMightCauseHorizontallyOverlappingBubbles(group)) {
```

Comment:
It surprises me that we need this for the new case but not for the two pre-existing situations where onFocusOnly is true. A comment would probably be appropriate (unless indeed the other cases do need it).